docs/guide/migration.md

```diff
@@ -28,6 +28,8 @@ There are other breaking changes which only affect few users.
   - [`build.cssMinify`](/config/build-options#build-cssminify) is now enabled by default even for SSR builds.
 - [[#18209] refactor!: bump minimal terser version to 5.16.0](https://github.com/vitejs/vite/pull/18209)
   - Minimal supported terser version for [`build.minify: 'terser'`](/config/build-options#build-minify) was bumped to 5.16.0 from 5.4.0.
+- [[#18231] chore(deps): update dependency @rollup/plugin-commonjs to v28](https://github.com/vitejs/vite/pull/18231)
+  - [`commonjsOptions.strictRequires`](https://github.com/rollup/plugins/blob/master/packages/commonjs/README.md#strictrequires) is now `true` by default (was `'auto'` before). The top-level `this` was incorrectly replaced with `globalThis` before, now it is correctly replaced with `module.exports`.
```

Comment:
Should the ``The top-level `this`...`` part be another list item? Though I think we could probably remove it altogether as I feel like it's more of a bug fix. The `strictRequires` change is more impactful I think.

Reply:
I put it because it was marked as breaking change, but was wondering if this can be removed. Let's remove it 👍

```suggestion
  - [`commonjsOptions.strictRequires`](https://github.com/rollup/plugins/blob/master/packages/commonjs/README.md#strictrequires) is now `true` by default (was `'auto'` before).
```